src/signin.js

```diff
@@ -0,0 +1,82 @@
+import { ERROR_MSG, REGEX_EMAIL, USER_INFO } from './constant.js';
+
+const email = document.getElementById('email');
+const password = document.getElementById('password');
+const signinForm = document.querySelector('.sign-signin-form');
+const eyeIcon = document.querySelector('.eye');
+
+//Check email: 입력 여부 확인, 메일 형식 확인-------------------------------
+const errorEmail = document.createElement('div');
+errorEmail.className = 'error-text';
+
+function checkEmail() {
+  if (!email.value) {
+    showInputError(email, errorEmail, ERROR_MSG.INPUT_EMAIL);
+  } else if (validateEmail(email.value)) {
+    showInputError(email, errorEmail, ERROR_MSG.INPUT_EMAIL_ADDRESS);
+  } else {
+    errorEmail.remove();
+    email.classList.remove('error-input');
+  }
```

Comment:
if-else if- else 구문이 길어지면 가독성울 해치게 됩니다. 요럴 때 [early return](https://devscb.tistory.com/130)을 사용해보심 좋아요 

```suggestion
  if (!email.value) {
    showInputError(email, errorEmail, ERROR_MSG.INPUT_EMAIL);
    return;
  } 
  if (validateEmail(email.value)) {
    showInputError(email, errorEmail, 
  ERROR_MSG.INPUT_EMAIL_ADDRESS);
  return;
  }

    errorEmail.remove();
    email.classList.remove('error-input');

```

Reply:
7496ff6f0295c587ff523cd08d08d767c25f4fb7 early return 문으로 수정했습니다 !